test/gtest/ucp/test_ucp_mmap.cc

```diff
@@ -424,16 +423,16 @@ UCS_TEST_P(test_ucp_mmap, reg) {
     }
 }
 
-UCS_TEST_P(test_ucp_mmap, reg_mem_type) {
+UCS_TEST_SKIP_COND_P(test_ucp_mmap, reg_mem_type,
+                     mem_buffer::supported_mem_types().size() == 1) {
```

Comment:
why?

Reply:
there is a test above which does the same, but for HOST-only

Reply:
can we combine them to one then?

Reply:
> can we combine them to one then?

that test checks registering of several host buffers.
this test checks registering of several host/cuda/rocm buffers selected randomly.

Reply:
sounds like this test covers the previous one which can be deleted

Reply:
done for registration and allocation tests